doc/src/Howto_rheo.rst

```diff
@@ -0,0 +1,115 @@
+Reproducing hydrodynamics and elastic objects (RHEO)
+====================================================
+
+The RHEO package is a hybrid implementation of smoothed particle
+hydrodynamics (SPH) for fluid flow, coupled to the :doc:`BPM package <Howto_bpm>`
```

Comment:
Does this mean the RHEO package would never be used by itself?  Only in conjunction with BPM ?

Reply:
It can still be used by itself to run a pure-fluid simulation (e.g. the Taylor-Green example). But it still needs to be built with BPM.

Reply:
I clarified the language